__mocks__/react-native.js

```diff
@@ -1,7 +1,8 @@
+/* eslint-disable */
```

Comment:
Because I declare multiple React components in one file.
But I will use `/* eslint-disable react/no-multi-comp */`.